src/components/InfoText.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+import styled from 'styled-components'
+
+interface Props {
+  title: string,
+  description: string
+}
+
+function InfoText({title, description}: Props) {
```

Comment:
Use the FC generic type provided react types to type your components

Reply:
Done